docs/src/components/mobile-nav.tsx

```diff
@@ -72,9 +71,6 @@ export function MobileDropdown(props: {
             </div>
           ))}
         </ScrollArea>
-        <div className="border-t pt-4">
-          <ThemeToggle />
-        </div>
```

Comment:
where is this now?

Reply:
It's already in the header

![screely-1689577467091](https://github.com/t3-oss/t3-env/assets/109742866/46dcaf3b-52b4-4b61-8be0-95106b2b729f)